spark-plugins/src/test/java/co/cask/hydrator/plugin/spark/test/SparkPluginTest.java

```diff
@@ -124,7 +126,8 @@ public static void setupTest() throws Exception {
     addPluginArtifact(NamespaceId.DEFAULT.artifact("spark-plugins", "1.0.0"), parents,
                       NaiveBayesTrainer.class, NaiveBayesClassifier.class,
                       KafkaStreamingSource.class, KafkaUtils.class, DefaultDecoder.class,
-                      TwitterStreamingSource.class);
+                      TwitterStreamingSource.class, InitialPositionInStream.class,
```

Comment:
This will just add the artifact where is the unit test which tests the KinesisStreamingSource ?